README.md

```diff
@@ -15,88 +15,36 @@ If you use `pip`, you can install it as:
 pip install aiida-cp2k
 ```
 
-
-# Features
```

Comment:
I really like such lists of features in the readme - I believe it is typically a sign of good design if what a python package does can be explained succinctly in a few words, and I would suggest to keep it here.

One alternative could be to keep the README absolutely minimal (nothing more than a pointer to the docs) and move everything to the docs. Up to you, but whatever you decide, I would suggest to clearly define for yourself what type of content should go in the README and what in the docs.

Reply:
I will keep it minimal. I think this way is much better.